test/Record.purs

```diff
@@ -121,7 +122,7 @@ instance traverseProp ::
   ) =>
   FoldingWithIndex
     (TraverseProp f k)
-    (SProxy sym)
+    (Proxy sym)
```

Comment:
I wasn't sure why `SProxy` didn't work here, so I swapped it to `Proxy`

Reply:
It doesn't work because HFoldlWithIndex uses Proxy.